[2023-04-26T09:50:31.029+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag1.retrieve_data manual__2023-04-26T09:50:30.102248+00:00 [queued]>
[2023-04-26T09:50:31.036+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dag1.retrieve_data manual__2023-04-26T09:50:30.102248+00:00 [queued]>
[2023-04-26T09:50:31.036+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T09:50:31.037+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T09:50:31.039+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T09:50:31.049+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): retrieve_data> on 2023-04-26 09:50:30.102248+00:00
[2023-04-26T09:50:31.056+0000] {standard_task_runner.py:55} INFO - Started process 4179 to run task
[2023-04-26T09:50:31.060+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'retrieve_data', 'manual__2023-04-26T09:50:30.102248+00:00', '--job-id', '251', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp9jip3c90']
[2023-04-26T09:50:31.061+0000] {standard_task_runner.py:83} INFO - Job 251: Subtask retrieve_data
[2023-04-26T09:50:31.136+0000] {task_command.py:376} INFO - Running <TaskInstance: dag1.retrieve_data manual__2023-04-26T09:50:30.102248+00:00 [running]> on host c452347a64f2
[2023-04-26T09:50:31.194+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=retrieve_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T09:50:30.102248+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T09:50:30.102248+00:00
[2023-04-26T09:50:31.204+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-04-26T09:50:31.211+0000] {sql.py:315} INFO - Running statement: select first_name, email from employees;, parameters: None
[2023-04-26T09:50:31.214+0000] {sql.py:324} INFO - Rows affected: 6
[2023-04-26T09:50:31.226+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=retrieve_data, execution_date=20230426T095030, start_date=20230426T095031, end_date=20230426T095031
[2023-04-26T09:50:31.280+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T09:50:31.305+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
